app/views/welcome/show.html.erb

```diff
@@ -8,9 +8,10 @@
     </div>
     <p class="fs-4">Long term preservation of scholarly works at Stanford</p>
     <div class="text-center text-md-left way-down">
-      <h2 class="mb-5">Make a deposit today!</h2>
+      <h2>Go to the SDR Dashboard</h2>
+      <h4 class="mb-4">to create or manage deposits</h4>
```

Comment:
where is h3? For accessibility reasons you shouldn't skip header levels

Reply:
Didn't realize this was an accessibility problem - I had made it an h4 since it better matched the sizing shown in the design, but I switched it to an h3 (it's not that far off).